Extensions/DataSources/src/DataSourcesHub/DataSourcesHub.tsx

```diff
@@ -0,0 +1,23 @@
+import "./DataSourcesHub.scss";
```

Comment:
In general, in overall code I see uneven usage of semicolons.
somewhere they are used and somewhere they are not used.
As in TS they are not mandatory so your code is working but we should stick to one method. (I prefer using semicolons)

Please go through all code and fix it.